docs/releases/2.0.txt

```diff
@@ -305,6 +301,11 @@ Miscellaneous
 * :class:`~django.views.generic.base.RedirectView` no longer silences
   ``NoReverseMatch`` if the ``pattern_name`` doesn't exist.
 
+* :class:`~django.forms.FloatField` and :class:`~django.forms.DecimalField`
+  now respect :setting:`DECIMAL_SEPARATOR` and :setting:`THOUSAND_SEPARATOR`
+  during validation correctly when :setting:`USE_L10N` is off.
```

Comment:
Sorry @claudep I got somehow confused and forgot there was an example in your comment. I think the example needs to be a bit different (``LANGUAGE_CODE`` is ignored if ``USE_I10N`` is off), but I proposed one now in the patch.